doc/Makefile

```diff
@@ -52,7 +52,7 @@ clean:
 	rm -rf $(BUILDDIR)/*
 
 html:
-	$(SPHINXBUILD) -W -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
```

Comment:
please revert this change. It's important to fail on warnings since this is what happens on CI.

Use `make develop` which does the thing that you want.

Reply:
My bad. Didn't intentionally commit this

Reply:
Also, didn't know about `make develop`. Thanks for pointing that out